### PR TITLE
[DR-OCR] Add missing modifier to sendRequest in oracle contract

### PR DESCRIPTION
**Pull request by bolekk**

### CODE REVIEW

**Review by pinebit — APPROVED** [review]
Wait for @justinkaseman approval as well.

**Review by KuphJr — APPROVED** [review]

**Review by justinkaseman — APPROVED** [review]
LGTM, good catch!

**Pull request merged**
